### DISCUSSION TITLE
.ImportErrorWhenRunningHook: Failed to import module __PyInstaller_hooks_0_PyQt6

### DISCUSSION THREAD

**Discussion by ImmanuelT8** [pyinstaller/pyinstaller#7648] [Help]
I get this error:

" .ImportErrorWhenRunningHook: Failed to import module __PyInstaller_hooks_0_PyQt6 required by hook for module C:\Users\emanu\miniconda34\lib\site-packages\PyInstaller\hooks\hook-PyQt6.py. Please check whether module __PyInstaller_hooks_0_PyQt6 actually exists and whether the hook is compatible with your version of C:\Users\emanu\miniconda34\lib\site-packages\PyInstaller\hooks\hook-PyQt6.py: You might want to read more about hooks in the manual and provide a pull-request to improve PyInstaller." 

Can someone help?

**Comment by rokm** [comment]
Not without the *complete* error message (i.e., the part that came before this).
